contracts/solidity/foundry.toml

```diff
@@ -5,7 +5,7 @@ libs = ["lib"]
 gas_reports = ["*"]
 
 [rpc_endpoints]
-goerli = "${GOERLI_RPC_URL}"
+goerli = "${ETH_RPC_URL}"
```

Comment:
If we are going to be generic for multiple chains `goerli` variable should be called network. Pls check what's the standard to administrate deployment for testnet + main net. Someone must have solved this problem in the past.

Reply:
From what I could see, they handle things [with profiles](https://book.getfoundry.sh/reference/config/overview?highlight=toml#configuration-format); I'm going to look for some examples

Reply:
We can improve this in the next PR. This could be merged by just changing there the name to something more generic and we do it properly later on